configure.ac

```diff
@@ -1,6 +1,17 @@
 AC_PREREQ([2.68])
 AC_INIT([rkt], [0.7.0+git], [https://github.com/coreos/rkt/issues])
 
+dnl if version ends with +git, append a short git-hash.
+AS_IF([test `expr match 'AC_PACKAGE_VERSION' '.*+git$'` -gt 0],
+      dnl version has +git suffix
+      [RKT_VERSION="AC_PACKAGE_VERSION`git rev-parse --short HEAD`"],
```

Comment:
We should handle that better (basically ignore the git error) because I think the right thing to do there is set version to "0.7.0+git" and that's what we're doing anyway.